app/views/layouts/blacklight/base.html.erb

```diff
@@ -46,7 +46,11 @@
     <div class="ucb_container">
         <%= render partial: 'shared/header_navbar' %>
         <main id="main-container" class="<%= container_classes %>" role="main" aria-label="<%= t('blacklight.main.aria.main_container') %>">
-            <%= content_for(:container_header) %>
+            <span class="constraints-label sr-only visually-hidden"><%= t('blacklight.search.filters.title') %></span>
+            <%= content_for(:container_header) || content_tag(:div, class: "sr-only visually-hidden-focusable") do
+                content_tag(:h1, "Main Content") +
+                content_tag(:h2, "Section Content")
+            end %>
```

Comment:
1. Should not be marked 'focusable'.
2. The h2 is unnecessary.
3. The h1 should describe the page, not just say "main content"; since this is a fallback, it can be the name of the site.

```suggestion
            <%= content_for(:container_header) || content_tag(:div, class: "sr-only visually-hidden") do
                content_tag(:h1, "UC Berkeley Library GeoData")
            end %>
```